README.md

```diff
@@ -219,6 +219,7 @@ Bucket order is signficant: Quota is determined for a `HIT` request by finding t
 
 The following optional fields are also supported:
 
+* `label`: A short, slug-like name for the rule. If set, Prometheus metrics for hits matching the rule will be labeled with `rule_label` as this value.
```

Comment:
I think it would be helpful to call out explicitly that this value is ignored for statsd metrics. We can consider adding support for additional statsd counters on a per-label basis down the road?

Reply:
done!